src/pilot/templates/roles_data.yaml

```diff
@@ -0,0 +1,200 @@
+# Specifies which roles (groups of nodes) will be deployed
```

Comment:
Should put the standard license header at the top of this file.

Reply:
Done.